.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
this only create and publish images if a release has been created.

We could change this to create images on other actions, but I thought this would be a nice minimum action to have as a starting point at least.

Reply:
that more than enough.